swap/src/bin/swap.rs

```diff
@@ -222,8 +170,135 @@ async fn main() -> Result<()> {
             // Print the table to stdout
             table.printstd();
         }
-        Options::Resume { .. } => todo!("implement this"),
+        Command::Resume {
+            swap_id,
+            bitcoind_url,
+            bitcoin_wallet_name,
+            monero_wallet_rpc_url,
+            listen_addr,
+        } => {
+            let db_swap = db.get_state(swap_id)?;
```

Comment:
There is a `resume_from_database` function. We should either use it here or delete it.

Reply:
True, I did mention that in the comments. Removing it would mean that we have to add loading the state to the `e2e` tests. I can do that.